tools/unicode_properties_parse/grapheme_break_property_data_gen.py

```diff
@@ -34,6 +34,26 @@ def parsePropertyLine(inputLine: str) -> Optional[PropertyRange]:
     else:
         return None
 
+def compactPropertyRange(input: list[PropertyRange]) -> list[PropertyRange]:
+    """
+    Merges consecutive ranges with the same property to one range.
+
+    Merging the ranges results in less ranges in the output table. Reducing the
+    size of the binary and improving the performance of the loookup.

```

Comment:
Grammar/spelling:

* "fewer ranges" (because they're countable)
* "Reducing ..." is a sentence fragment.
* "loookup" needs to be slightly compacted.

Suggestion: "binary size" and "lookup performance" would be less verbose.